test/test_model.py

```diff
@@ -969,6 +969,28 @@ def test_run_error(self):
         self.assertEqual(str(cm.exception), 'ERROR cannot get status\n')
         self.assertEqual(cm.exception.args[0], 'ERROR cannot get status\n')
 
+    @patch("pwd.getpwuid")
+    @patch("grp.getgrgid")
+    def test_push_path_unnamed(self, getpwuid: MagicMock, getgrgid: MagicMock):
+        getpwuid.side_effect = KeyError
+        getgrgid.side_effect = KeyError
+        harness = ops.testing.Harness(ops.CharmBase, meta='''
+            name: test-app
+            containers:
+              foo:
+                resource: foo-image
+            ''')
+        harness.begin()
+        harness.set_can_connect('foo', True)
+        c = harness.model.unit.containers['foo']
+
+        with tempfile.TemporaryDirectory() as push_src:
+            push_path = pathlib.Path(push_src) / 'src.txt'
+            with push_path.open('w') as f:
+                f.write('hello')
```

Comment:
Could use `Path.write_text` here?
```suggestion
            push_path.write_text('hello')
```

Reply:
TIL :)